### ISSUE TITLE
Mega-menu links should be a single unordered list

### ISSUE THREAD

**Issue by shawnbot** [uswds/uswds#1884]
As mentioned in #1508, our header navigation mega-menus are awkward for screen readers because they break up subnavigation into multiple unordered lists (one list per visual column). We should investigate using [CSS multicolumn layouts](http://caniuse.com/#feat=multicolumn), supported in IE10+ and all other modern browsers.

> HP Header - Unnecessary list attributes are used, Each of the submenu items should be coded in one nested list to accurately convey the parent/child hierarchy of the item. All logically related items must be coded in one unordered list structure. List items_ExtendedHeader.png. Currently, these links are coded in in 3 or 4 unordered lists and the child elements are not being properly nested beneath the parent elements. this is confusing screen reader users. #1884

**Comment by maya** [comment]
Each megamenu is using a single list so maybe this was previously resolved. Closing.

**Comment by maya** [comment]
Re-opening with additional details... 

While the mega-menu is a single list, it includes a `div` as the child of a `ul` which causes an HTML5 error:

>Element 'div' not allowed as child element in this context.

CSS multicolumn layout would work for the current design and once we drop IE9 support. But would not work for a design of columns with headings that need to be broken up with different numbers of links per columns. For example:

Policies (heading)
- link 1

Benefits (heading)
- link 1
- link 2
- link 3

Contact (heading)
- link 1
- link 2

For the above, it would seem like it would need to support multiple lists.

**Comment by maya** [comment]
Your suggestion is appreciated, however it is not on the current product roadmap. 

Note: we can change the example to 3 discrete lists in the meantime.
